README.md

```diff
@@ -1 +1,35 @@
-# ucw-app
\ No newline at end of file
+# Universal Connect Widget (Application)
+
+This repo is a full stack application which anyone can clone and self-host as a way to serve the connect widget via a url which can then be loaded into an iframe.
```

Comment:
My only question here is whether or not the final widget package will serve a url. If we go the route of exposing a React Component, then it won't work like this. 

If we go the route of using docker to serve the widget url, or use `single-spa.js` to serve the frontend, then it could continue to work as it originally did.

Reply:
I think either way it'll load from a url because the way it's implemented currently is to hit the server url with an authCode and the server loads the front-end widget. So whether it's loaded via a microfrontend or react component it will still have to be imported into the front-end part of the full stack ucw-app. I think the decision to use react or a microfrontend only really impacts users who want to use the widget alone in their app and build their own backend for it.